include/download.mk

```diff
@@ -13,7 +13,7 @@ ifdef PKG_SOURCE_VERSION
     PKG_VERSION := $(if $(PKG_SOURCE_DATE),$(subst -,.,$(PKG_SOURCE_DATE)),0)~$(call version_abbrev,$(PKG_SOURCE_VERSION))
   endif
   PKG_SOURCE_SUBDIR ?= $(PKG_NAME)-$(PKG_VERSION)
-  PKG_SOURCE ?= $(PKG_SOURCE_SUBDIR).tar.xz
+  PKG_SOURCE ?= $(PKG_SOURCE_SUBDIR).zst
```

Comment:
.tar.zst?

Reply:
Yeah, exactly as otherwise detection will break and ZSTD tarballs are not extracted

Reply:
Feel free to change this. I saw .zst at other places so I thought it's the way to go.

Reply:
I also notice for zst .tar is not used... Gnu tar checks the extension to detect a tar archive?

Reply:
No idea, but without piping it via zstdcat it wont get extracted.